### PR TITLE
Move checkstyle checks to Presto

### PR DESCRIPTION
**Pull request by tdcmeehan**
It's more flexible to leave them in Presto.

### CODE REVIEW

**Comment by mayankgarg1990** [comment]
How would this work for presto-facebook?

**Comment by wenleix** [comment]
Checkstyle in Presto is removed in https://github.com/prestodb/presto/pull/9658 without mentioning removing checkstyle in commit message/PR test, etc.

File: pom.xml [hunk 1]
```diff
@@ -39,6 +39,7 @@
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
         <air.check.skip-jacoco>true</air.check.skip-jacoco>
+        <air.checkstyle.config-file>src/checkstyle/presto-checks.xml</air.checkstyle.config-file>
```
Comment (wenleix) [inline pom.xml]: Curious: how does this work? -- will it still use the checkstye file in airlift?  (So we sort of have double check here? )

Comment (tdcmeehan) [reply pom.xml]: It's overriding the location of the checkstyle configuration file location from airbase to one we specify in Presto.  I tested this by locally adding some incompatible rules and I see corresponding failures in the child modules that weren't due to rules found in airbase.

**Pull request merged**